modules/vue-pinia/concepts/VuePiniaPlugins.js

```diff
@@ -0,0 +1,13 @@
+//@ts-check
+
+import { GeneratingConcept } from '@storefront-x/core'
+
+export default class VuePiniaPlugin extends GeneratingConcept {
+  get directory() {
+    return 'vuePinia/plugins'
+  }
+
+  get exportAll() {
```

Comment:
I dont think you need exportAll here. It was for vue plugins because of its after lifecycle hook.